### PR TITLE
export getDeviceSIlently

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: lib/services/devices/deviceService.js [hunk 1]
```diff
@@ -1228,6 +1228,7 @@ function retrieveDevice(deviceId, apiKey, callback) {
 exports.listDevices = intoTrans(context, checkRegistry)(listDevices);
 exports.listDevicesWithType = intoTrans(context, checkRegistry)(listDevicesWithType);
 exports.getDevice = intoTrans(context, checkRegistry)(getDevice);
+exports.getDeviceSilently = intoTrans(context, checkRegistry)(getDeviceSilently);
```
Comment (fgalan) [inline lib/services/devices/deviceService.js:1231]: No CNR entry needed?

Comment (AlvaroVega) [reply lib/services/devices/deviceService.js:1231]: No CNR update is needed, it's just reopen https://github.com/telefonicaid/iotagent-node-lib/pull/891

Comment (fgalan) [reply lib/services/devices/deviceService.js:1231]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**